services/license.js

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert');
+
+function validationError(attribute) {
+  return `License: Invalid "${attribute}" provided`;
+}
+
+function assertParams({ id, name, groups, compatibility, regexp, url }) {
+  assert(typeof id === 'string' && id.length > 0, validationError('id'));
+  assert(typeof name === 'string' && name.length > 0, validationError('id'));
```

Comment:
Looks like there are some copy-paste leftovers: should be `validationError('name')` here I think -> and respectively the other attribute names for later ones ✌️